data/json/items/comestibles/med.json

```diff
@@ -1167,6 +1167,29 @@
                 [ "calcium", 75 ]
             ]
         }
+    },
+        {
+        "id" : "bonemeal_tablet",
+        "type" : "COMESTIBLE",
+        "comestible_type" : "MED",
+        "name" : "bone meal tablet",
+        "description" : "Homemade calcium supplement made out of bone meal. Tastes horribly and is hard to swallow but it does its job.",
+        "weight" : 1,
+        "volume" : 1,
+        "price" : 10,
+        "fun" : -8,
```

Comment:
This fun is reasonable for an ash and bone pill, but I think a flavored bonemeal tablet with fun: -1 or so is also reasonable.